pkg/cmd/cli/install/install.go

```diff
@@ -86,6 +86,9 @@ type Options struct {
 	ScheduleSkipImmediately         bool
 	PodResources                    kubeutil.PodResources
 	KeepLatestMaintenanceJobs       int
+	BackupRepoConfig                string
```

Comment:
Maybe we should update the vars here as well:
- `BackuprepoConfig -> BackupRepoConfigMap`
- `RepoMaintenanceJobConfig -> RepoMaintenanceJobConfigMap`
Same as we are doing for `NodeAgentConfigMap`

Reply:
Modified.